pkg/plugin/processor/standalone/host_module.go

```diff
@@ -145,3 +153,101 @@ func (m *hostModuleInstance) commandResponse(ctx context.Context, buf types.Byte
 	m.commandResponses <- tuple[*processorv1.CommandResponse, error]{&resp, nil}
 	return 0
 }
+
+// createSchema is the exported function that is called by the WASM module to
+// create a schema and set the buffer bytes to the marshalled response.
+// It returns the response size on success, or an error code on error.
+func (m *hostModuleInstance) createSchema(ctx context.Context, buf types.Bytes) types.Uint32 {
+	m.logger.Trace(ctx).Msg("executing create_schema")
+
+	var req processorv1.CreateSchemaRequest
+	var resp processorv1.CreateSchemaResponse
+	length := len(buf)
+	if m.parkedCreateSchemaRequest != nil {
```

Comment:
We don't need to park the create schema request, the processor will always send us the whole request, as it knows exactly how big it is. However, the processor has no idea about how big the response will be, so we might need to park the response.

That said, we need to store the request bytes (or a hash + size, but that might be overkill), so that if we have a parked response we can compare the requests and see if the processor actually repeated the same request.

Pseudocode for how I think this should work:
```
var buf types.Bytes
if parkedResponse != nil {
    if buf[:len(lastRequest)] == lastRequest {
        clear parkedResponse and lastRequest for next call
        return parkedResponse
    }
}
unmarshal request
response := schemaService.CreateSchema(unmarshalledRequest)
if len(response) > len(buf) {
    parkedResponse = response
    copy(lastRequest, buf)
    return len(response)
}
marshal response into buf
return len(response)
```

We could create a generic utility for this stuff so we can reuse it in all host functions.